test/commands/test_alias.rb

```diff
@@ -1,7 +1,48 @@
 require 'minitest/autorun'
+require_relative '../test__helper'
+require_relative '../fake_home'
+require_relative '../../lib/zold/commands/alias'
 
-# @todo #279:30min Write unit tests for the alias command.
-#  It should test all the subcommands (set, remove and show)
-#  and any options.
 class TestAlias < Minitest::Test
+  # alias set <wallet> <alias>
+  def test_set_writes_alias_to_the_alias_file
+    skip
+    FakeHome.new.run do |home|
+      wallet = home.create_wallet
+      Zold::Alias.new(wallets: home.wallets, log: test_log).run(%W[set #{wallet.id} my-alias])
+      assert_equal read_alias_file(home), %W[my-alias #{wallet.id}]
```

Comment:
@botanicus a string would probably do here?

Reply:
@ledestin how about:

```
My-alias 1234567890
Another-alias 1234567890
```

Vs:

```
My-alias         1234567890
Another-alias 1234567890
```

Reply:
@ledestin I'm saying the test should not be whitespace sensitive.